gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -38,7 +38,10 @@ def open(vm_disk)
     (Get-Item "#{@virtual_disk}").length
 STAT_EOL
     file_size, stderr = @parser.parse_single_powershell_value(run_correct_powershell(stat_script))
-    raise "Unable to obtain virtual disk size for #{vm_disk}" if stderr.include?("At line:")
+    if @network && stderr.include?("RegisterTaskDefinition")
+      raise "Unable to obtain virtual disk size for #{vm_disk}. Check Hyper-V Host Domain Credentials"
+    end
+    raise "Unable to obtain virtual disk size for #{vm_disk}" unless stderr.empty?
```

Comment:
@roliveri @djberg96 I've added code with an array of possible error strings to be checked.  Currently populated with the two cases I've called out above.  Pushed and ready to go.